tests/console/sshd.pm

```diff
@@ -207,16 +211,22 @@ sub check_journal {
 
 sub post_run_hook {
     my $self = shift;
-    check_journal();
+    self->cleanup();
     $self->SUPER::post_run_hook;
 }
 
 sub post_fail_hook {
     my $self = shift;
-    check_journal();
+    self->cleanup();
```

Comment:
I wonder if the `check_journal();` in cleanup might not cause some problems here. If that ones fails, it might swallow the previous error message.

Minor issue though, feel free to ignore it

Reply:
I have no idea how failures inside the post hooks are treated. Do we need a `post_post_fail_hook_fail_hook`? :P

Reply:
For now I just swapped the order.